exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -38,7 +38,7 @@ public bool DoesGameHaveMinimumPlayersRequired()
             return (howManyPlayers() >= 2);
         }
 
-        public bool add(string playerName)
+        public bool GeneratePlayerList(string playerName)
```

Comment:
Could have called this AddPlayer as you're adding a player at a time. But good spot that just calling it "add" was ambiguous 👍